app/src/main/java/com/spendster/presentation/editExpenses/EditExpensesActivity.java

```diff
@@ -1,16 +1,60 @@
 package com.spendster.presentation.editExpenses;
 
 import android.os.Bundle;
+import android.view.View;
+import android.widget.EditText;
+import android.widget.TextView;
 
+import com.google.gson.Gson;
 import com.spendster.R;
+import com.spendster.data.entity.Expense;
+import com.spendster.presentation.utils.SDate;
+
+import java.util.Date;
 
 import androidx.appcompat.app.AppCompatActivity;
 
-public class EditExpensesActivity extends AppCompatActivity {
+public class EditExpensesActivity extends AppCompatActivity implements View.OnClickListener {
+
+    private TextView tvTitle;
+    private TextView tvCategory;
+    private EditText etAmount;
+    private TextView tvToday;
+    private EditText etNote;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_add_expenses);
+        unitUI();
+        setValues();
+    }
+
+    private void unitUI() {
+        this.etNote = findViewById(R.id.etNote);
+        this.tvTitle = findViewById(R.id.tvTitle);
+        this.tvCategory = findViewById(R.id.tvCategory);
+        this.etAmount = findViewById(R.id.etAmount);
+        this.tvToday = findViewById(R.id.tvToday);
+    }
+
+    private void setValues(){
+        Expense expense = getExpense();
```

Comment:
@SimonMaha It's better to pass expense as parameter, and pass getExpense() function from there